src/hammer-tech/hammer_tech.py

```diff
@@ -1120,6 +1124,40 @@ def get_tech_pcb_hooks(self, tool_name: str) -> List['HammerToolHookAction']:
         """
         return list()
 
+    def extract_to_cache(self,extract_list:List[str]) -> List[str]:
```

Comment:
Would like to have seen `extract_tarballs` use this method too. You'd need to check for `.tar` after the `gzip` to do the untar-ing, but this is fine for now.